lib.ftd

```diff
@@ -139,8 +139,47 @@ ftd.image-src src:
 boolean $hover-me: false
 
 -- ftd.column:
-width.fixed.percent if { ftd.device != "mobile" }: 49
-width.fixed.percent if { ftd.device == "mobile" }: 100
+width: fill-container
+
+-- ftd.column:
+if: { ftd.device != "mobile" }
+width.fixed.percent: 49
+;;width.fixed.percent if { ftd.device == "mobile" }: 100
+$on-mouse-enter$: $ftd.set-bool($a = $ad-card.hover-me, v = true)
+$on-mouse-leave$: $ftd.set-bool($a = $ad-card.hover-me, v = false)
+border-bottom-width.px if {$ad-card.hover-me}: 11
```

Comment:
Use { ad-card.hover-me } instead of {$ad-card.hover-me}. No need for $ inside if conditions. Do this for any other such cases as well.

Reply:
@Heulitig `ftd.set-bool` function need `mutable` variable that's why its used, so can't change it.